lib/stripy.ex

```diff
@@ -25,10 +25,20 @@ defmodule Stripy do
   """
 
   @doc "Constructs HTTPoison header list with auth."
-  def headers(%{secret_key: sk, version: v}) do
-    [{"Authorization", "Bearer #{sk}"},
-     {"Content-Type", "application/x-www-form-urlencoded"},
-     {"Stripe-Version", v}]
+  def headers(params) do
```

Comment:
Would be good to expand the `@doc` for `headers/1` with the list of accepted keys for the headers.